sky_alert/endpoint.py

```diff
@@ -1,22 +1,22 @@
+import requests
 from fastapi import FastAPI
 import os
 from dotenv import load_dotenv
-import requests
 import datetime
 from sky_alert.protocol import SunData
-from typing import Optional, Any, Union
+from typing import Any, Union
 
 load_dotenv()
 
 app = FastAPI()
 
 
-@app.get("/healthz")  # type: ignore[misc]
+@app.get("/healthz")  # type: ignore
```

Comment:
This can't be removed because of this: https://stackoverflow.com/questions/75347974/mypy-untyped-decorator-makes-function-main-untyped-for-fastapi-routes

I'll add a comment here until this issue gets fixed.